# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
# Commit: 2a373d864092
# Full Hash: 2a373d864092cdd0eb970b247e27f320650ec2c7
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 9
# Description:
#   Bug 1654112 - Rename MediaSessionConduit::DeleteStreams to Shutdown, as VideoConduit does more than just delete streams. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112856
# ==============================================================================

diff -r 09db50c8f370 -r 2a373d864092 dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
--- a/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Wed Apr 14 17:36:40 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Wed Apr 14 17:40:24 2021 +0200
@@ -47,7 +47,7 @@
                                         std::move(aStsThread));
 }
 
-void WebrtcAudioConduit::DeleteStreams() {
+void WebrtcAudioConduit::Shutdown() {
   MOZ_ASSERT(mCallThread->IsOnCurrentThread());
   MutexAutoLock lock(mMutex);
   DeleteSendStream();